lib/ecmarkdown.d.ts

```diff
@@ -244,6 +242,8 @@ declare namespace Parser {
     location?: { pos: number; end: number };
```

Comment:
This needs to be updated to match the definition.
```suggestion
    location?: { 
      start: { line: number; column: number, offset: number };
      end: { line: number; column: number, offset: number };
    };
```

Reply:
Oops, yup. I knew I was going to forget these.

I'm going to migrate this project to TypeScript soon, I think, so maybe we can just rebase this PR on top of that one and have it tell us all these things.